### PR TITLE
fix/fiat-price-value

### PR DESCRIPTION
**Pull request by leozera14**
Fix fiat price value in full application screens and implement some validations to format the value)

### CODE REVIEW

**Review by lucasgabrielgsp — APPROVED** [review]

File: source/hooks/usePrice.ts [hunk 1]
```diff
@@ -23,11 +23,20 @@ export const usePrice = () => {
...
+
+    const fractionValidation =
+      quantityOfZerosAfterDot !== -0 && quantityOfZerosAfterDot <= 6;
+
     const currencySymbol = getSymbolFromCurrency(currency);
 
     const formattedValue = value.toLocaleString(navigator.language, {
-      minimumFractionDigits: precision,
-      maximumFractionDigits: precision,
+      minimumFractionDigits: fractionValidation
```
Comment (claudiovb) [inline source/hooks/usePrice.ts:34]: Pls, double-check this section of the code. There is an erratic behavior when switching from network to network, that causes pali UI to break  with the following error, the Home component is the one to break due to minimumFractionDigit value being out of range during the network switch:
image

**Review by claudiovb — CHANGES_REQUESTED** [review]
The PR works as intended the fiat prices are being displayed properly. Apart from the bug mentioned at the usePrice hooks, everything else seems to be good

File: source/hooks/usePrice.ts (hunk 1 above)
Comment (leozera14) [reply source/hooks/usePrice.ts]: Issue already fixed in new commits

**Comment by claudiovb** [comment]
image
The price for syscoin NEVM is not being fetched. The relationship is 1 to 1 so just getting the price from syscoin as its being done on UTXO chain should be fine

**Pull request merged**